met-api/migrations/versions/779d9125c8cb_add_reviewer_type.py

```diff
@@ -0,0 +1,33 @@
+"""add_reviewer_type
+
+Revision ID: 779d9125c8cb
+Revises: 8def759e43d9
+Create Date: 2023-06-12 16:26:29.300751
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '779d9125c8cb'
+down_revision = '8def759e43d9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('membership', 'type', type_=sa.Text())
+    op.execute('DROP TYPE membershiptype;')
+    op.execute('CREATE TYPE membershiptype AS ENUM (\'TEAM_MEMBER\', \'REVIEWER\');')  
+    op.alter_column('membership', 'type', type_=sa.Enum('TEAM_MEMBER', 'REVIEWER', name='membershiptype'), postgresql_using='type::membershiptype')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('membership', 'type',
```

Comment:
how is the membershiptype downgraded here?

Reply:
Just downgrading the column, keeping the existing type.